src/neuroglancer/datasource/graphene/frontend.ts

```diff
@@ -61,6 +65,7 @@ import {Uint64} from 'neuroglancer/util/uint64';
 import {makeDeleteButton} from 'neuroglancer/widget/delete_button';
 import {DependentViewContext} from 'neuroglancer/widget/dependent_view_widget';
 import {makeIcon} from 'neuroglancer/widget/icon';
+import {CancellationToken, CancellationTokenSource} from 'src/neuroglancer/util/cancellation';
```

Comment:
fix src/neuroglancer/ path to neuroglancer/

Reply:
Done, I'm also going to put up a pull request to fix some other imports.